### ISSUE TITLE
Ichimuku

### ISSUE THREAD

**Issue by denouezechias** [arun-mansa/tradingbot#5]
Hi !
please can you add ichimuko in the indicator group

**Comment by tahmi123** [comment]
install ta-lib

**Comment by denouezechias** [comment]
yes, i have already see ta-lib and all this indicators. but how can i "plug" it on your project becausei tried to do it but unfortunally, i get error

**Comment by tahmi123** [comment]
`    def call(self):
        """Method to check call pattern."""
        logger = logging.getLogger("__main__")
        candles = self.candles

        if hasattr(candles, 'candles_array'):
            candel_array = candles.candles_array
            close = pd.Series([candle.candle_close for candle in  candel_array])
            open = pd.Series([candle.candle_open for candle in  candel_array])
            high = pd.Series([candle.candle_high for candle in  candel_array])
            low = pd.Series([candle.candle_low for candle in  candel_array])
            up, lw, MA = talib.BBANDS(close, matype=MA_Type.T3)
            rsi14 = talib.RSI(close, timeperiod=14)`
so I edited my tbh.py pattern to that.
dont forget to import talib

**Comment by denouezechias** [comment]
yes, i already see the tbh.py file
thanks very very much for this code !

**Comment by tahmi123** [comment]
you welcome

**Comment by denouezechias** [comment]
please, can you give me the link of the ta-lib ? it seem that i have is the wrong talib and there are many ta-lib  on github

**Comment by tahmi123** [comment]
mehn installing ta-lib takes the special grace of God and i mean that in every sense, try "https://github.com/mrjbq7/ta-lib"

**Comment by denouezechias** [comment]
ichimuko is not in the list of indicators.
please, i need you really. i tried to modify the code to add ichimuku but at the start, i need only ssb and kijun code.

**Comment by tahmi123** [comment]
that's the best i've got man
